### PR TITLE
React index

### PR DESCRIPTION
**Pull request by ChrisDonohue0628**
created a react component for the index page, letting all of the museums from our api being added dynamically and then introducing pagination through a previous and next page button

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -280,7 +317,11 @@ DEPENDENCIES
   tzinfo-data
   uglifier (>= 1.3.0)
   valid_attribute
+<<<<<<< HEAD
+  yelp
+=======
   web-console
+>>>>>>> master
```
Comment (kyjoya) [inline Gemfile.lock]: Merge conflict here!

File: app/views/museums/index.html.erb [hunk 2]
```diff
@@ -12,3 +16,4 @@
 <% @museums.each do |museum| %>
 <%= link_to museum.name, museum_path(museum)%>
 <% end %>
+>>>>>>> master
```
Comment (kyjoya) [inline app/views/museums/index.html.erb]: Merge conflict

File: config/routes.rb [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
+
+  resources :museums do
+=======
   devise_for :users, controllers: {
     sessions: 'users/sessions'
   }
   root 'museums#index'
   namespace :admin do
     resources :museums
+>>>>>>> master
```
Comment (kyjoya) [inline config/routes.rb]: merge conflict

File: db/seeds.rb [hunk 4]
```diff
@@ -1,3 +1,27 @@
...
+  Museum.create!(
+  name: museum['name'],
+  location: museum['formatted_address'],
+  hours: museum['opening_hours']['weekday_text'].join('\n'),
+  website: museum['website'],
+  phone_number: museum['formatted_phone_number'],
+  rating: result['rating']
+  )
+end
+=======
```
Comment (kyjoya) [inline db/seeds.rb]: merge conflict

File: react/src/main.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import 'babel-polyfill';
+import React from 'react';
```
Comment (ChrisDonohue0628) [inline react/src/main.js]: needed for react pages

File: react/src/main.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+import ReactDOM from 'react-dom';
+import MuseumsContainer from './components/MuseumsContainer'
```
Comment (ChrisDonohue0628) [inline react/src/main.js]: needed for react render

File: react/src/components/MuseumsContainer.js [hunk 7]
```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import Museum from './Museum';
```
Comment (ChrisDonohue0628) [inline react/src/components/MuseumsContainer.js]: Needed for React Rendering

File: react/src/components/MuseumsContainer.js [hunk 8]
```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
```
Comment (ChrisDonohue0628) [inline react/src/components/MuseumsContainer.js]: Needed for React Rendering

**Pull request merged**